pkg/api/resource/quantity_test.go

```diff
@@ -667,6 +731,22 @@ func TestSub(t *testing.T) {
 	}
 }
 
+func TestNeg(t *testing.T) {
+	tests := []struct {
+		a        Quantity
+		expected Quantity
+	}{
+		{Quantity{dec(10, 0), DecimalSI}, Quantity{dec(-10, 0), DecimalSI}},
+	}
+
+	for i, test := range tests {
+		test.a.Neg(test.a)
```

Comment:
In addition to performing it on itself, test it on a nil, a 0, and a non-zero number.